core/dbt/version.py

```diff
@@ -44,16 +48,26 @@ def get_version_information():
 
     plugin_version_msg = "Plugins:\n"
     for plugin_name, version in _get_dbt_plugins_info():
-        plugin_version_msg += '  - {plugin_name}: {version}\n'.format(
-            plugin_name=plugin_name, version=version
-        )
+        version_major = dbt.semver.VersionSpecifier.from_version_string(version).major
+        plugin_update_msg = ''
+        if installed.major == version_major:
+            compatibility_msg = green('compatible')
+        else:
+            compatibility_msg = red('requires update!')
+            plugin_update_msg = ("  To update dbt-{} please run:\n"
+                                 "    pip install dbt-{} --upgrade\n\n".format(plugin_name, plugin_name))
```

Comment:
Love the really actionable feedback here! My only comment is that not all users install dbt via pip - I really like the idea of making the instructions easy, but it'd be confusing to give pip instructions to someone who installed dbt via Homebrew etc.

Not sure how to square that circle; perhaps we have to fall back to linking them to the [docs](https://docs.getdbt.com/dbt-cli/install/overview)?

Reply:
Looks like the pip message is provided as an example in the original feature request. But I agree with @joellabes 's fallback to docs idea here. It would make later updates easier also.

Reply:
Hi @ChenyuLInx @joellabes 

Ok, then simply we need to put a link of related docs page, right?
Besides the yellow one will be added also.

Thanks.

Reply:
That would be perfect! Thanks @nkyuray